code/datums/abilities/ability_parent.dm

```diff
@@ -198,11 +198,7 @@
 			owner.attach_hud(hud)
 
 	proc/Stat()
```

Comment:
then delete it?

Reply:
Removed - I'd left it in because it was still required for for revenant health but that's now been removed by #11709.

Atomisation issue 😅